be/src/olap/page_cache.h

```diff
@@ -103,21 +103,22 @@ class StoragePageCache {
     class DataPageCache : public LRUCachePolicy {
     public:
         DataPageCache(size_t capacity, uint32_t num_shards)
-                : LRUCachePolicy("DataPageCache", capacity, LRUCacheType::SIZE,
+                : LRUCachePolicy(CachePolicy::Type::DATA_PAGE_CACHE, capacity, LRUCacheType::SIZE,
```

Comment:
rename to CACHE_TYPE
CachePolicy::TYPE  user may treat it as policy type. But actually, it is one kind of cache, not one kink of policy like lru,fifio...

Reply:
done